deployments/helm-chart/README.md

```diff
@@ -217,6 +217,10 @@ Parameter | Description | Default
 `controller.service.httpsPort.targetPort` | The target port of the HTTPS port of the Ingress Controller service. | 443
 `controller.serviceAccount.name` | The name of the service account of the Ingress Controller pods. Used for RBAC. | Autogenerated
 `controller.serviceAccount.imagePullSecretName` | The name of the secret containing docker registry credentials. Secret must exist in the same namespace as the helm release. | ""
+`controller.serviceMonitor.create` | Create a ServiceMonitor custom resource. | false
```

Comment:
ca you add the `name` here and in `values.yaml`?

Reply:
Done!  Thanks for the feedback!